docs/learn/beginner/00-app-anatomy.md

```diff
@@ -196,7 +196,7 @@ Each module should also implement the `RegisterServices` method as part of the [
 
 ### gRPC `Query` Services
 
-gRPC `Query` services allow users to query the state using [gRPC](https://grpc.io). They are enabled by default, and can be configured under the `grpc.enable` and `grpc.address` fields inside [`app.toml`](../../user/run-node/01-run-node.md#configuring-the-node-using-apptoml-and-configtoml).
+gRPC `Query` services allow users to query the state using [gRPC](https://grpc.io). They are enabled by default, and can be configured under the `grpc.enable` and `grpc.address` fields inside [`app.toml`](https://docs.cosmos.network/main/user/run-node/run-node).
```

Comment:
btw, changing those links by direct links to GitHub will make the UX in the website weird, as it will redirect there instead of opening properly on the doc website.

Reply:
We actually dont keep the `user` section in this repo anymore. Its in the `sdk-docs-repo` repo so maybe i need to put a disable here as it is pulling from the `sdk-docs-repo` anyways so the link isnt technically broken

Reply:
Yeah, I agree what matters most is that they aren't broken when built in sdk-docs-repo, not here.